core/server/worker/src/main/java/alluxio/worker/block/MonoBlockStore.java

```diff
@@ -266,6 +280,8 @@ public void updatePinnedInodes(Set<Long> inodes) {
 
   @Override
   public void registerBlockStoreEventListener(BlockStoreEventListener listener) {
+    LOG.debug("registerBlockStoreEventListener: listener={}", listener);
+    mBlockStoreEventListeners.add(listener);
```

Comment:
now we have two places manage these listeners, which is vague. Considering keeping single reference (e.g. in the local blockstore) and in the mono block store you can get the reference from the local block store, instead of using its own one.